operations/deployment/crunchy-postgres/templates/PostgresCluster.yaml

```diff
@@ -0,0 +1,184 @@
+          memory: {{ .Values.instances.limits.memory }}
+      sidecars:
+        replicaCertCopy:
+          resources:
+            requests:
+              cpu: {{ .Values.instances.sidecars.replicaCertCopy.requests.cpu }}
+              memory: {{ .Values.instances.sidecars.replicaCertCopy.requests.memory }}
+            limits:
+              cpu: {{ .Values.instances.sidecars.replicaCertCopy.limits.cpu }}
+              memory: {{ .Values.instances.sidecars.replicaCertCopy.limits.memory }}
+      dataVolumeClaimSpec:
+        accessModes:
+          - "ReadWriteOnce"
+        resources:
+          requests:
+            storage: {{ .Values.dataVolumeClaimSpec.storage }}
+        storageClassName: netapp-block-standard
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 1
+              podAffinityTerm:
+                topologyKey: kubernetes.io/hostname
+                labelSelector:
+                  matchLabels:
+                    postgres-operator.crunchydata.com/cluster:
+                      {{ template "crunchy-postgres.fullname" . }}
+                    postgres-operator.crunchydata.com/instance-set: {{ .Values.instances.name }}-ha
+  users:
+    - name: {{ template "crunchy-postgres.fullname" . }}
+      databases:
+        - {{ template "crunchy-postgres.fullname" . }}
+      options: "CREATEROLE"
+    - name: postgres
+      databases:
+        - {{ template "crunchy-postgres.fullname" . }}
+  backups:
+    pgbackrest:
+      image: {{ .Values.pgbackrest.image }}
+      global:
+        repo1-retention-full: {{ .Values.pgbackrest.retention | quote }}
+      repos:
+        - name: {{ .Values.pgbackrest.repos.name }}
+          schedules:
+            full: {{ .Values.pgbackrest.repos.schedules.full }}
+            incremental: {{ .Values.pgbackrest.repos.schedules.incremental }}
+          volume:
+            volumeClaimSpec:
+              accessModes:
+                - {{ .Values.pgbackrest.repos.volume.accessModes }}
+              resources:
+                requests:
+                  storage: {{ .Values.pgbackrest.repos.volume.storage }}
+              storageClassName:
+                {{ .Values.pgbackrest.repos.volume.storageClassName }}
+          
+          # Storage buckets backup
+          {{- if .Values.pgbackrest.repos.s3.enabled }}
+          s3:
+            bucket: {{ .Values.pgbackrest.repos.s3.bucket | quote }}
+            endpoint: {{ .Values.pgbackrest.repos.s3.endpoint | quote }}
+            region: {{ .Values.pgbackrest.repos.s3.s3.region | quote }}
+          {{- end }}
+
+          {{- if .Values.pgbackrest.repos.gcs.enabled }}
+          gcs:
+            bucket: {{ .Values.pgbackrest.repos.gcs.bucket | quote }}
+          {{- end }}
+
+          {{- if .Values.pgbackrest.repos.azure.enabled }}
+          azure:
+            container: {{ .Values.pgbackrest.repos.azure.container | quote }}
+          {{- end }}
```

Comment:
`repos` is also a list. It's possible to set up several different repos with different endpoints / schedules. This configuration constrains us to one repo. This could become pretty large to live in a values.yaml file though. I wonder if we could include templates and ingest them here with .Files.Get?
https://helm.sh/docs/chart_template_guide/accessing_files/